src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs

```diff
@@ -47,6 +48,7 @@ public async Task<IActionResult> AddAndApprove(BulkUploadAddAndApproveDraftAppre
                 new BulkUploadAddAndApproveDraftApprenticeshipsCommand
                 {
                     ProviderId = request.ProviderId,
+                    FileUploadLogId = request.FileUploadLogId,
```

Comment:
I can't quite see any test for this new parameter

Reply:
Added param in existing test